go.mod

```diff
@@ -5,13 +5,12 @@ go 1.24.0
 require (
 	github.com/hashicorp/go-hclog v1.6.3
 	github.com/hashicorp/go-tfe v1.92.0
+	github.com/hashicorp/jsonapi v1.5.0
 	github.com/mitchellh/cli v1.1.5
 	github.com/sethvargo/go-retry v0.3.0
 	go.uber.org/mock v0.6.0
 )
 
-require golang.org/x/sync v0.17.0 // indirect
```

Comment:
Since go 1.17, there should be two `require` blocks, the first with direct dependencies and the second with indirect https://go.dev/doc/go1.17#go-command